.drone.yml

```
@@ -1,124 +1,138 @@
-      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DB
+      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DATABASEHOST
       - cd ../server
-      - php ./build/signed-off-checker.php
-    secrets: [ github_token ]
-    when:
-      matrix:
-        TESTS: signed-off-check
-  check-app-compatbility:
-    image: nextcloudci/php7.1:php7.1-16
-    environment:
-      - APP_NAME=data_request
-      - CORE_BRANCH=master
-      - DB=sqlite
-    commands:
-        # Pre-setup steps
-        - wget https://raw.githubusercontent.com/nextcloud/travis_ci/master/before_install.sh
-        - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DB
-        - cd ../server
-
-        # Code checker
-        - ./occ app:check-code $APP_NAME
-        - cd apps/$APP_NAME/
-    when:
-      matrix:
-        TESTS: check-app-compatbility
-  syntax-php7.1:
-    image: nextcloudci/php7.1:php7.1-16
+      - composer install
+      - ./lib/composer/bin/parallel-lint apps/$APP_NAME/
+  - name: syntax-php7.3
+    image: nextcloudci/php7.3:php7.3-1
     environment:
-      - APP_NAME=data_request
-      - CORE_BRANCH=master
-      - DB=sqlite
+      APP_NAME: data_request
+      CORE_BRANCH: stable16
+      DATABASEHOST: sqlite
     commands:
-      # Pre-setup steps
       - wget https://raw.githubusercontent.com/nextcloud/travis_ci/master/before_install.sh
-      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DB
+      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DATABASEHOST
       - cd ../server
       - composer install
       - ./lib/composer/bin/parallel-lint apps/$APP_NAME/
-    when:
-      matrix:
-        TESTS: syntax-php7.1
-  php7.1:
+  - name: app-code-check
+    image: nextcloudci/php7.3:php7.3-1
+    environment:
+      APP_NAME: data_request
+      CORE_BRANCH: stable16
+      DATABASEHOST: sqlite
+    commands:
+      - wget https://raw.githubusercontent.com/nextcloud/travis_ci/master/before_install.sh
+      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DATABASEHOST
+      - cd ../server
+      - ./occ app:check-code $APP_NAME
+      - cd apps/$APP_NAME/
+
+trigger:
+  branch:
+    - master
+    - stable*
+  event:
+    - pull_request
+    - push
+
+---
+kind: pipeline
+name: unit-sqlite-php7.1
+
+steps:
+  - name: sqlite-php7.1
     image: nextcloudci/php7.1:php7.1-16
     environment:
-      - APP_NAME=data_request
-      - CORE_BRANCH=master
-      - DB=sqlite
+      APP_NAME: data_request
+      CORE_BRANCH: stable16
+      DATABASEHOST: sqlite
     commands:
-      # Pre-setup steps
       - wget https://raw.githubusercontent.com/nextcloud/travis_ci/master/before_install.sh
-      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DB
-      - cd ../server/
+      - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DATABASEHOST
+      - cd ../server
       - ./occ app:enable $APP_NAME
       - cd apps/$APP_NAME
 
       # Run phpunit tests
-      - cd tests/unit/
+      - cd tests/php/
```

Shouldn't it be this?

```suggestion
      - cd tests/unit/
```